charts/ingress/ingress.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingestion
+  annotations:
+    kubernetes.io/ingress.class: azure/application-gateway
+    appgw.ingress.kubernetes.io/backend-path-prefix: "/"
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /ingestion/*
+            pathType: Prefix
+            backend:
+              service:
+                name: ingestion-service
```

Comment:
In our terraform in the playground repo, we are naming these services "phdi-{env}-ingestion-service", etc. We'll need to make sure they match for this to work. I think it's okay to remove the env from the service names since they are already namespaced to the cluster. That way we won't have to do any fancy interpolation here. We'll need to adjust the terraform in the other repo, though.

Reply:
Ahh makes sense, will update em! Thanks Nick